.travis.yml

```diff
@@ -2,12 +2,27 @@ language: python
 cache: pip
 dist: xenial
 
-matrix:
```

Comment:
Could you explain why this movesfrom matrix to jobs?

Reply:
I meant in the commit message, not here :)

Reply:
To move this into different stages to we dont build the docs if there is a failure on the tests, mostly it gains us time and we can reuse it later for extra steps that we dont necessarily want to run unless the linting is passed first (like unit tests that take longer? molecule testing?)

Kind of like the jenkins pipeline, if step X fails do not move into the next one.

Reply:
oh, that makes sense....